dev-client/src/theme.ts

```diff
@@ -190,16 +190,6 @@ export const theme = extendTheme({
           width: 'full',
           justifyContent: 'flex-start',
         },
-        confirmModal: {
```

Comment:
Question: was this dead code? I don't see it replicated anywhere else

Reply:
this was one of the things i was curious to ask your opinion about re: refactoring the components to be more re-usable/separate styling/content concerns

when a specific component wants to have its own bespoke styling, i was thinking it makes more sense to define those styles next to the component rather than in the global theme file. the global theme file would be for style values that need to be re-used across many components

Reply:
but open to feedback here!